src/init.cpp

```diff
@@ -71,13 +71,15 @@
 #include <zmq/zmqrpc.h>
 #endif
 
+namespace {
 bool fFeeEstimatesInitialized = false;
 static const bool DEFAULT_PROXYRANDOMIZE = true;
```

Comment:
I think you could drop `static`?

Reply:
Yes, absolutely but I didn't want to change any existing lines to make review super trivial. Let me know if that decision is blocking an utACK from you and I'll consider reconsidering :-)

Reply:
With or without the declaration seems they'll have internal linkage (via unnamed namespace) and static storage duration (via namespace scope).

Mildly in favor of removal on the grounds that a distinction without a difference in this case can be misleading.

> `static` When used in a declaration of an object, it specifies static storage duration. When used in a declaration at namespace scope, it specifies internal linkage.
> static storage duration. All objects declared at namespace scope (including global namespace) have this storage duration, plus those declared with static or extern.
https://en.cppreference.com/w/cpp/language/storage_duration

> Unnamed namespace definition. Its members have potential scope from their point of declaration to the end of the translation unit, and have internal linkage.
https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces